build.ps1

```diff
@@ -74,7 +74,7 @@ if (Test-Path "$RepoRoot\artifacts")
 
 mkdir "$RepoRoot\artifacts" | Out-Null
 
-$DOTNET_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1"
+$DOTNET_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain/dotnet-install.ps1"
```

Comment:
Why are you changing this to master at this point? If you do this, you are forcing the templating repo to move to 2.0, which I am not sure they want to do now. Though, they might.